src/Umbraco.Community.Contentment/DataEditors/DataList/DataSources/XmlDataListSource.cs

```diff
@@ -154,5 +174,35 @@ public XmlNotesConfigurationField()
 </div>", true)
             { }
         }
+
+
+        /// <summary>
+        /// Returns an instance of XmlNamespaceManager with a specified XmlDocument.NameTable.  
+        /// Returns null if no namespace is defined.
+        /// </summary>
+        /// <param name="doc">The XmlDocument</param>
+        /// <returns>XmlNamespaceManager if there is at least one namespace, null if there is no namespace defined.</returns>
+        private XmlNamespaceManager CreateNamespaceManager(XmlDocument doc)
```

Comment:
I'd like this method higher up in the code file. I tend to put any private nested classes (e.g. in this case, `XmlNotesConfigurationField`), towards the bottom.